webhooks.go

```diff
@@ -89,14 +90,17 @@ func (c *Client) VerifyWebhookSignature(ctx context.Context, httpReq *http.Reque
 		TransmissionSig  string          `json:"transmission_sig,omitempty"`
 		TransmissionTime string          `json:"transmission_time,omitempty"`
 		WebhookID        string          `json:"webhook_id,omitempty"`
-		Event            json.RawMessage `json:"webhook_event"`
+		Event            json.RawMessage `json:"webhook_event,omitempty"`
 	}
 
 	// Read the content
 	var bodyBytes []byte
 	if httpReq.Body != nil {
 		bodyBytes, _ = ioutil.ReadAll(httpReq.Body)
 	}
+	else{
```

Comment:
Please keep else on the same line with closing bracket and add a space before opening bracket.